erdpy_network/tokens.py

```diff
@@ -0,0 +1,98 @@
+from erdpy_network.interface import IAddress
+from erdpy_core import Address
+from typing import Any, List, Dict
+from erdpy_network.primitives import Nonce
+
+
+class FungibleTokenOfAccountOnNetwork:
+    def __init__(self):
+        self.identifier: str = ''
+        self.balance: int = 0
+        self.raw_response: Any = {}
```

Comment:
I see, we also keep the original response here. Seems all right. Let's think if we should do this for all the other objects, as well - in the future.